ranger.tf

```diff
@@ -153,14 +149,9 @@ resource "aws_lb_target_group" "ranger_admin_tg" {
   depends_on = ["aws_lb.ranger_admin_lb"]
 }
 
-data "vault_generic_secret" "ranger_admin_cert" {
-  path = "${local.vault_path}/${aws_route53_record.ranger_admin.fqdn}"
-}
-
-resource "aws_iam_server_certificate" "ranger_admin" {
-  name             = "ranger-admin"
-  certificate_body = "${data.vault_generic_secret.ranger_admin_cert.data["crt"]}"
-  private_key      = "${data.vault_generic_secret.ranger_admin_cert.data["pem"]}"
+data "aws_iam_server_certificate" "ranger_admin" {
```

Comment:
Does this mean that any TF apps using this module now need to create this IAM SSL cert as part of their own terraform?  If so, should put something in the CHANGELOG or README to specifically call this out.

Reply:
@barnharts4  - this is a data source, so it will fetch the details of the existing resource which matches `name` as `"${aws_route53_record.ranger_admin.fqdn}"` .
@rpoluri  - where is `aws_iam_server_certificate` created ?

Reply:
The intent is to manage this outside TF module, it can be in wrapper code or can be added manually. Added this to notes section in README